test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -8958,6 +8958,78 @@ public void RestoreNetCore_PackagesLockFile_WithProjectChangeFramweorksAndLocked
             }
         }
 
+
+
+        [Fact]
+        public async void RestoreNetCore_PackagesLockFile_WithReorderedRuntimesInLockFile_PassRestore() { 
+            // A project with RestoreLockedMode should pass restore if the runtimes in the lock file have been reordered
+            using (var pathContext = new SimpleTestPathContext())
+            {
+
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
```

Comment:
I think we should set the runtime identifiers explicitly. It will be a more robust test if we do that.

Reply:
See changes